rpcpb/rpc.proto

```diff
@@ -681,6 +692,12 @@ message LoadSnapshotRequest {
   optional string log_root_dir = 11;
   // wallet private key
   string wallet_private_key = 12;
+  // custom network genesis path
```

Comment:
I would avoid setting this. the snapshot should contain this data and not seeing a user case so strong to enabling changin it here.

Reply:
same reason I did not added upgrade path to this section

Reply:
agree